fabric-item-api-v1/build.gradle

```diff
@@ -1,7 +1,26 @@
 version = getSubprojectVersion(project)
 
-moduleDependencies(project, ['fabric-api-base'])
+moduleDependencies(project, [
+	'fabric-api-base',
+	'fabric-registry-sync-v0'
+])
 
 testDependencies(project, [
 	':fabric-content-registries-v0',
+	':fabric-data-generation-api-v1'
 ])
+
+loom {
```

Comment:
`fabricApi { configureDataGeneration() }` does not working for sub modules or something?

Reply:
Is that a new loom feature? I just copied from the datagen gradle file.

Reply:
Ah, then it's okay

> Is that a new loom feature?
Yes

Reply:
Id keep them all the same. I can look into moving to the new loom feature everwhere in a seperate PR.